chart/Chart.yaml

```diff
@@ -47,23 +47,23 @@ annotations:
       url: https://airflow.apache.org/docs/helm-chart/1.8.0/
   artifacthub.io/screenshots: |
     - title: DAGs View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/dags.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.7.0/_images/dags.png
     - title: Datasets View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/datasets.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.7.0/_images/datasets.png
     - title: Grid View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/grid.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.7.0/_images/grid.png
     - title: Graph View
-      url: https://airflow.apache.org/docs/apache-airflow/2.6.3/_images/graph.png
+      url: https://airflow.apache.org/docs/apache-airflow/2.7.0/_images/graph.png
```

Comment:
I checked... the new graph and gantt images have the same filename.

Reply:
I don't fully follow. Both resolves to different filename(for the links here)

Reply:
I just mean, generally, we can't always just update the version in the url. Both graph and gantt were refactored in 2.7, so there was a possibility the new screenshot had a different filename. But they were updated this time around.

This was meant for future reviewers, just wanted to note that I already checked that these 1) exist and 2) are the right images.